docs/stats.md

```diff
@@ -655,6 +655,95 @@ framework which has the same calling conventions as the single site stats,
 we can rework the sections above.
 :::
 
+### Two-locus site statistics
+
+The {meth}`~TreeSequence.ld_matrix` method provides access to a collection of
+two-locus statistics. These statistics are computed on a site-by-site basis and
+are designed to operate on multi-allelic data. Within this framework, we also
+implement polarisation for certain statistics requiring polarisation.
+
+Because we implement two-locus statistics for multi-allelic data, we need a
+method for combining the statistical results from each allele into one summary
+for a pair of sites. We have two normalisation methods for combining these
+results:
+
+`hap_weighted`
+: {math}`\sum_{i=1}^{n}\sum_{j=1}^{m}p(A_{i}B_{j})F_{ij}`
+  where {math}`F` is the summary function and {math}`p(A_{i}B_{j})` is the
+  frequency of haplotype {math}`A_{i}B_{j}`. This method was first introduced in
+  [Karlin (1981)](https://doi.org/10.1111/j.1469-1809.1981.tb00308.x), and was
+  reviewed in [Zhao (2007)](https://doi.org/10.1017/S0016672307008634).
+
+  This method multiplies each result by the frequency of the haplotype under
+  consideration (the {math}`[i,j]` alleles for a given pair of sites), then sums
+  the results to obtain the statistic for the two focal sites.
+
+`total_weighted`
+: {math}`\frac{1}{(n-\mathbb{1}_{p}) (m-\mathbb{1}_{p})}\sum_{i=1}^{n}\sum_{j=1}^{m}F_{ij}`
+
+  where {math}`\mathbb{1}_{p}` is an indicator function conditioned on whether
+  or not our statistic is polarized, {math}`n` is the number of alleles in site
+  {math}`a`, and {math}`m` is the number of alleles in site {math}`b`.
+
+  This method assigns equal weight to each of the possible haplotypes for a
+  given pair of sites.
+
+Generally speaking, we use the `hap_weighted` method of normalisation for
+unpolarised ratio statistics, such as {math}`r^2` and {math}`D^{\prime}`.
+We use `total_weighted` to normalise the rest.
+
+The two-locus statistics all take frequencies of haplotype configurations as
+input. The function signature below ({math}`f(f_a, f_b, f_{ab})`) describes a
```

Comment:
Small thing on style: you use "f" for both function and allele/haplotype frequencies. Above, you use capital F for the summary function - could that be used here, to help with readability?